catkin_ws/src/arena_camera/src/arena_camera_parameter.cpp

```diff
@@ -365,6 +370,18 @@ void ArenaCameraParameter::readFromRosParameterServer(const ros::NodeHandle& nh)
   nh.param<double>("exposure_search_timeout", exposure_search_timeout_, 5.);
   nh.param<double>("auto_exposure_upper_limit", auto_exp_upper_lim_, 10000000.);
 
+  if (nh.hasParam("enable_aoi"))
+  {
+    nh.getParam("enable_aoi", enable_aoi_);
+  }
```

Comment:
i'd add a space between statements like this to help readability (avoid looking like else if at a glance)